Blockzilla/UIConstants.swift

```diff
@@ -233,6 +233,7 @@ struct UIConstants {
         static let aboutSafariBullet1 = NSLocalizedString("About.safariBullet1", value: "Block trackers for improved privacy", comment: "Label on About screen")
         static let aboutSafariBullet2 = NSLocalizedString("About.safariBullet2", value: "Block Web fonts to reduce page size", comment: "Label on About screen")
         static let whatsNewTitle = NSLocalizedString("Settings.whatsNewTitle", value: "What’s New", comment: "Title for What's new screen")
+        static let sumoTopicWhatsNew = NSLocalizedString("Settings.whatsNew", value: "whats-new-focus-ios-7", comment: "URL for what's new in this release")
```

Comment:
This does not need to be localized, please include it at the bottom of this file (near ```shareTrackersTipTitle```) and not as an NSLocalizedString